lib/recurly/bus.js

```diff
@@ -77,12 +77,13 @@ export class Bus extends Emitter {
   /**
    * Resets the bus
    *  - Empties recipients
-   *  - disconnects listener
+   *  - cycles listener
    *
    * @public
    */
   reset () {
     this.recipients = [];
     window.removeEventListener('message', this.receive, false);
+    connect ();
```

Comment:
Oh, the shame of it all...

Must've knocked it off while reviewing the diff. Sorry!